src/main/java/de/unistuttgart/towercrushbackend/service/websockets/GameService.java

```diff
@@ -420,10 +419,10 @@ private void saveAndDeleteGame(final Game game) {
      *
      * @param sleepTime sleep time
      */
-    private void sleep(final int sleepTime) {
+    private void sleep(final int sleepTime){
         try {
             Thread.sleep(sleepTime);
-        } catch (final InterruptedException e) {
+        } catch (final InterruptedException e) { //NOSONAR
```

Comment:
```suggestion
        } catch (final InterruptedException e) {
```
Shouldn't be needed here.